definitions/src/lib.rs

```diff
@@ -412,3 +412,10 @@ pub enum RowStatus {
     SQL_ROW_ERROR = 5,
     SQL_ROW_SUCCESS_WITH_INFO = 6,
 }
+
+#[repr(C)]
+pub struct LibmongosqltranslateDataIO {
```

Comment:
This name is a little bit too specific if you consider what it represents.
Could we use something more generic? Like ByteBuffer or ByteArray or RawBson?